src/content/docs/en/guides/assets.mdx

```diff
@@ -204,6 +204,34 @@ const allBlogPosts = await getCollection("blog");
 }
 ```
 
+### Allowed domains and remote patterns for remote images
+
+For security reasons, Astro will not process remote images from all domains by default. You can specify which domains to allow in your `astro.config.mjs` file.
+
+```js title="astro.config.mjs"
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+  image: {
+    domains: ["astro.build"],
+  }
+});
+```
+
+Alternatively, you can specify a pattern to match against the `src` attribute of remote images to allow. This can be useful if you want to allow images from a CDN that uses a wildcard subdomain.
+
+```js title="astro.config.mjs"
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+  image: {
+    remotePatterns: [{ hostname: "*.akamaihd.net" }],
+  }
+});
+```
+
+The protocol, hostname, pathname, and port of the image URL can be checked using this method. For instance, you could allow only images being served from HTTPs using `{ protocol: "https" }`
```

Comment:
```suggestion
The protocol, hostname, pathname, and port of the image URL can be checked using this method. For instance, you could allow only images being served from HTTPs using `{ protocol: "https" }`.

See [`image.domains`](/en/reference/configuration-reference/#imagedomains) and [`image.remotePatterns`](/en/reference/configuration-reference/#imageremotepatterns) for detailed configuration instructions.
```
Note, these links will work once the accompanying config-reference docs are merged!